src/tools/rbd/action/Rename.cc

```diff
@@ -42,6 +42,7 @@ int execute(const po::variables_map &vm) {
     return r;
   }
 
+  std::string dst_pool;
   std::string dst_pool_name;
```

Comment:
Wouldn't you be able to just default ```dst_pool_name``` to the current value of ```pool_name```? If the pool name was provided in the positional or via the "--dest-pool" optional, it would be overwritten -- otherwise, since it isn't empty, it won't be defaulted to "rbd"